firmware/Core/Src/antenna_deploy_drivers/ant_commands.c

```diff
@@ -0,0 +1,68 @@
+#include "main.h"
+
+#include "antenna_deploy_drivers/ant_commands.h"
+#include "antenna_deploy_drivers/ant_internal_drivers.h"
+#include "stm32l4xx_hal_i2c.h"
+#include "debug_tools/debug_i2c.h"
+#include "debug_tools/debug_uart.h"
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdbool.h>
```

Comment:
Please avoid stdbool

Reply:
Removed that line to avoid using stdbool.